go-crud/books.json

```diff
@@ -0,0 +1,20 @@
+[
+  {
+    "Id": 1,
+    "Name": "The Bottled Leopard",
+    "Author": "Chukwuemeka Ike",
+    "PubData": {
+      "Year": 2020,
+      "Month": "March"
+    }
+  },
+  {
+    "Id": 2,
+    "Name": "Ajala's Calabash",
+    "Author": "Forgotten",
+    "PubData": {
+      "Year": 2007,
+      "Month": "March"
+    }
+  }
+]
```

Comment:
Always have a line feed at the end of the files. You can configure your editor to do this automatically.